lib/bitfield.h

```diff
@@ -72,7 +72,8 @@ DECLARE_MTYPE(BITFIELD);
 	do {                                                                   \
 		(v).n = 0;                                                     \
 		(v).m = ((N) / WORD_SIZE + 1);                                 \
-		(v).data = XCALLOC(MTYPE_BITFIELD, ((v).m * sizeof(word_t)));  \
+		(v).data = (word_t *)XCALLOC(MTYPE_BITFIELD,                   \
```

Comment:
casting is not needed why did you add it?

Reply:
removed